tests/integration/ha/continuous_writes.py

```diff
@@ -86,6 +85,7 @@ def count_znodes(parent: str, hosts: str, username: str, password: str) -> int:
     assert znodes
```

Comment:
**todo** this assert is strange. I mean: counting node and returning 0 is still fine for the function. We should put this assert in the tests, not in the function, like assert count_znodes(...)

Reply:
I'll just remove the line, it's not necessary.